docs/settings/reporting-settings.asciidoc

```diff
@@ -160,11 +155,9 @@ available, but there will likely be errors in the visualizations in the report.
   job, as many times as this setting. Defaults to `3`.
 
 | `xpack.reporting.capture.loadDelay`
-  | When visualizations are not evented, this is the amount of time before
-  taking a screenshot. All visualizations that ship with {kib} are evented, so this
-  setting should not have much effect. If you are seeing empty images instead of
-  visualizations, try increasing this value.
-  Defaults to `3000` (3 seconds).
+  | ({ref}/common-options.html#time-units[time value]) When visualizations are not evented, this is the amount of time before taking a
```

Comment:
To make this parallel with the other descriptions, you could use:

Specify the amount of time before taking a screenshot when visualizations are not evented.